std/conv.d

```diff
@@ -3275,7 +3275,20 @@ Target parse(Target, Source)(ref Source s)
     assert(b == true);
 }
 
-// input range to null literal conversions
```

Comment:
No public function should be undocumented. If the user is supposed to be able to use it, then they should know about it.